.github/workflows/deploy-runtime-installer-staging.yml

```diff
@@ -14,6 +17,5 @@ jobs:
       stage: "staging"
       version: 1
     secrets:
-      aws-access-key-id: ${{ secrets.ENCLAVES_PUBLIC_AWS_ACCESS_KEY_ID_STAGING }}
-      aws-secret-access-key: ${{ secrets.ENCLAVES_PUBLIC_AWS_ACCESS_SECRET_KEY_STAGING }}
+      AWS_ENCLAVES_OIDC_ROLE_ARN: ${{ secrets.AWS_ENCLAVES_STAGING_OIDC_ROLE_ARN }}
```

Comment:
tiny nit, but can you standardize the casing on the other secret? Can continue with the upper snake case